.github/workflows/validate_kong_image_trigger_via_label.yaml

```diff
@@ -14,6 +14,8 @@ jobs:
       ISSUE_NUMBER: ${{ github.event.issue.number }}
       GH_TOKEN: ${{ secrets.K8S_TEAM_BOT_GH_PAT }}
     steps:
+      - name: checkout repository
+        uses: actions/checkout@v4
```

Comment:
I think this job doesn't need to checkout the repo, right?

Reply:
This requires checkout of repo. If we do not checkout, the step will fail with the error 
```
failed to run git: fatal: not a git repository (or any of the parent directories): .git
```
.